eng/js.gdnbaselines

```diff
@@ -0,0 +1,44 @@
+{
+  "version": "latest",
+  "baselines": {
+    "baseline": {
+      "name": "baseline",
+      "createdDate": "2022-02-16 18:55:30Z",
+      "lastUpdatedDate": "2022-02-16 18:55:30Z"
+    }
+  },
+  "results": {
+    "edac2fc9c52db1bff42322ef6b7245c3a910af71715c7ff79b377ab6d38596b7": {
+      "signature": "edac2fc9c52db1bff42322ef6b7245c3a910af71715c7ff79b377ab6d38596b7",
+      "alternativeSignatures": [
+        "ea6eb0f83d5422b24185dbf915ecffb237bafb4a2beccc429379270d7a896fb3"
+      ],
+      "target": "sdk/identity/identity/test/public/browser/usernamePasswordCredential.spec.ts",
```

Comment:
Why do we need to bring back baseline file now? could we use the suppression file so we don't have two places to suppress errors?

Reply:
These are the one needs to remove finally. We updated the cred scan to a stable version. The one we currently use is legacy one. The stable version actually reports more errors than the stable version. 

As time being, we cannot wait to clean up and upgrade the version. So we put all in baseline file and work on cleaning up baseline.